src/indicators/ElementSizeIndicator.C

```diff
@@ -0,0 +1,44 @@
+// libMesh includes
+#include "libmesh/numeric_vector.h"
+
+// MOOSE includes
+#include "Assembly.h"
+#include "MooseVariable.h"
+#include "SystemBase.h"
+
+// Mastodon includes
+#include "ElementSizeIndicator.h"
+
+registerMooseObject("MastodonApp", ElementSizeIndicator);
+
+InputParameters
+ElementSizeIndicator::validParams()
+{
+  InputParameters params = Indicator::validParams();
+  params.addParam<MaterialPropertyName>(
+      "element_size",
+      "element_size",
+      "The value of the material that will control the mesh refinement.");
```

Comment:
This is the name of the property, and not the value. Please change the description. 

Also, make this a RequiredParam and remove the default.